distribution/lib/Standard/Base/0.0.0-dev/docs/api/Meta.md

```diff
@@ -26,9 +29,11 @@
     - methods self -> Standard.Base.Data.Vector.Vector
     - name self -> Standard.Base.Data.Text.Text
     - qualified_name self -> Standard.Base.Data.Text.Text
-- type Unresolved_Symbol
+- type Unresolved
```

Comment:
- @jdunkerley, how much do we care about compatibility?
- if we care, then I'll return back the name `Unresolved_Symbol`!

Reply:
- Let's integrate this PR as it is in de807c9
- e.g. with renaming `Unresolved_Symbol` to `Unresolved`
- btw, @Akirathan, an interesting way to _keep compatibility_ would be:

```ruby
export Unresolved as Unresolved_Symbol`
```

- that currently doesn't work, but it could I assume
- that way the `Unresolved` type would still remain visible as `Meta.Unresolved_Symbol`
- just its FQN would change 
   - which is acceptable incompatibility, I believe
   - as it only affects meta programming